web/src/routes/EventsPage.tsx

```diff
@@ -126,10 +146,43 @@ const EventsPage = ({ city, pathname, languageCode, cityCode }: CityRouteProps):
   if (event) {
     const { featuredImage, thumbnail, lastUpdate, content, title, location, date } = event
     const defaultThumbnail = featuredImage ? featuredImage.medium.url : thumbnail
-
-    const PageFooter = (
-      <StyledButton onClick={() => downloadEventAsIcsFile(event)} text={t('exportAsICal')} fullWidth={viewportSmall} />
-    )
+    const isRecurring = event.date.recurrenceRule && event.date.recurrenceRule.count() > 1
```

Comment:
Optional: Could we move the whole event export logic in this file to a separate component, e.g. EventExport or similar? I think that way we'd have less logic in this component, clearer separation of concern, possibility of reusing the event export and easier testing.